README.md

```diff
@@ -12,17 +13,11 @@ You will have to download wkhtmltopdf `0.12.x` in order to use Snappy.
 
 Please, check [FAQ](doc/faq.md) before opening a new issue. Snappy is a tiny wrapper around wkhtmltox, so lots of issues are already answered, resolved or wkhtmltox ones.
 
-[![Build Status](https://secure.travis-ci.org/KnpLabs/snappy.png?branch=master)](http://travis-ci.org/KnpLabs/snappy)
-
 Following integrations are available:
 * [`knplabs/knp-snappy-bundle`](https://github.com/KnpLabs/KnpSnappyBundle), for Symfony
 * [`barryvdh/laravel-snappy`](https://github.com/barryvdh/laravel-snappy), for Laravel
 * [`mvlabs/mvlabs-snappy`](https://github.com/mvlabs/MvlabsSnappy), for Zend Framework
 
-## Current maintainer(s)
-
-* [NiR-](https://github.com/NiR-)
```

Comment:
The idea was to let people know who's responsible for review/merge. If you will, it'd be better to put your github handle instead :wink:

Reply:
I removed it because we added the statement below at the end of the README and `Current maintainer(s)` section was a bit contradictory to it. 

I can add my GH handle at the end of it (as it has been done on https://github.com/KnpLabs/packagist-api for another maintainer).

```
Maintainers
KNPLabs is looking for maintainers (see why).

If you are interested, feel free to open a PR to ask to be added as a maintainer.

We’ll be glad to hear from you :)
```

Reply:
Oh ok, fine then :smile: